ci-operator/jobs/openshift/jenkins/openshift-jenkins-presubmits.yaml

```diff
@@ -29,8 +29,6 @@ presubmits:
         - name: PREPARE_COMMAND
           value: tag-in-image.sh
         - name: TEST_FOCUS
-          value: openshift pipieline
-        - name: TEST_FOCUS_SERIAL
```

Comment:
2) i don't think we need to pass serial at all, but i'll have to see what this does after i merge it.

Reply:
My prior experience was (remember my discovery that the test did not run in parallel) was
1) the TEST_FOCUS run ran 3 or 4 generic tests
2) the TEST_FOCUS_SERIAL ran the test under the `openshift pipeline` focus

If you are saying that removing TEST_FOCUS_SERIAL and just specifying TEST_FOCUS will result in 2) happening, and in parallel, then cool

If removing TEST_FOCUS_SERIAL only results in 1) happening, then yeah this doesn't do anything for us

Reply:
what tests do you want run with this job?  I only found two tests tagged with "openshift pipeline":

https://github.com/openshift/origin/blob/master/test/extended/builds/jenkins_plugin.go#L114
https://github.com/openshift/origin/blob/master/test/extended/builds/pipeline.go#L60

Neither of them need to be run serially.

I assume the reason "test focus" didn't run what you expected is because there was a typo in it.   I don't know why it would have run any generic tests, but we can see what this is running after it merges.

Reply:
totally missed the typo :-)

yes those are the tests I wanted to run

yes with the typo fixed, don't need serial ... though I was matching the pattern done for the image registry e2e test at the time but it has serial tests

so yeah remove

Reply:
the typo explains why I didn't get tests in parallel